libmpv/mpv.def

```diff
@@ -1,3 +1,5 @@
+__progname
+environ
```

Comment:
I absolutely don't see why libmpv should export these symbols. On the opposite, it seems broken and dangerous.

Reply:
In addition this leads to build failures on other OSes, see CI output.